Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -3959,6 +3961,24 @@ void CClientGame::IdleHandler ( void )
                 m_pManager->GetSoundManager ()->SetMinimizeMuted ( true );
         }
     }
+    if ( g_pCore->IsFocused() )
+    {
+        if (!m_bWasMinimized)
```

Comment:
You probably wanted to use `m_bWasFocus` instead (or `m_bWasFocused` if you rename it)